Tasks/AzureFileCopyV4/AzureUtilityARM.ps1

```diff
@@ -55,7 +55,7 @@ function Create-AzureContainer
         {
             $container = New-AzureStorageContainer -Name $containerName -Context $storageContext -ErrorAction Stop
         } else {
-            $container = New-AzureStorageContainer -Name $containerName -Context $storageContext -Permission Container -ErrorAction Stop
```

Comment:
Instead of if else, can we have single behaviour for all types of storage accounts? Also this issue is there for all versions of tasks not just v4

Reply:
This behavior is similar to AzureFileCopyV3, and we are making much changes in task side as of now. So don't think that we should change much code.
Morevoer, I could see the issue happening only in V4 version, let me know if you see this behavior in V3 also. The issue was raised for V4 also

Reply:
@AmrutaKawade Removed if else, since no specific behavior is there with Premium storage now